include/yaclib/coroutine/detail/via_awaiter.hpp

```diff
@@ -33,7 +33,7 @@ class ViaAwaiter : public ITask {
  private:
   enum class State : char { Empty = 0, HasCalled, HasCancelled };
 
-  IExecutorPtr _executor = nullptr;
+  IExecutorPtr _executor;
   yaclib_std::coroutine_handle<> _handle;
   IntrusivePtr<BaseCore> _promise;
```

Comment:
Why IntrusivePtr?

Reply:
It is more common interface inside details